fiatconnect-api.md

```diff
@@ -1796,6 +1796,897 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccountIfsc`
+
+`CheckingAccountIfsc` is a Fiat Account schema that represents accounts for transactions across `INR` currency.
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "ifsc": `string`,
+    "accountNumber": `string`,
+    "country": `string`,
+    "fiatAccountType": "CheckingAccountIfsc"
+}
+```
+####  7.3.2.3. `CheckingAccountIban`
+
+`CheckingAccountIban` is a Fiat Account schema that represents accounts for transactions across `EUR` currency.
```

Comment:
To my knowledge, it's actually an international standard not limited to Europe

Reply:
We will update this to a more generic statement for all countries that support IBAN account type format.

Reply:
@cajubelt - updated.